general/First.java

```diff
@@ -1,5 +1,5 @@
 class First {
     public static void main(String[] args) {
```

Comment:
no args,  use different name

Reply:
this is convention
please let us use this.